com.anysphere.Cursor.yml

```diff
@@ -6,41 +6,57 @@ command: zypak-run.sh
 base: org.electronjs.Electron2.BaseApp
 base-version: "24.08"
 finish-args:
-  # Basic X11 and Wayland access
-  - --socket=x11
+  - --share=ipc
   - --socket=wayland
+  - --socket=x11
   # GPU acceleration
   - --device=dri
   # Audio
   - --socket=pulseaudio
+  - --socket=ssh-auth
   # Network access - Cursor needs this for its AI features
   - --share=network
-  # Allow talking to the D-Bus session bus
-  - --socket=session-bus
-  - --socket=system-bus
+  - --device=all
+  - --filesystem=host
```

Comment:
The decision not to have devices nor host filesystem available was intentional for this flatpak. I feel that if the user feels comfortable giving more access to the filesystem, they should use flatseal. I prefer that by default it only has a workspace it creates.